contrib/babelfishpg_common/src/varbinary.c

```diff
@@ -1503,6 +1506,67 @@ varbinary_leq(PG_FUNCTION_ARGS)
 	PG_RETURN_BOOL(result);
 }
 
+Datum
+int4varbinary_div(PG_FUNCTION_ARGS)
+{
+       int32      source1 = PG_GETARG_INT32(0);
+       bytea      *source2 = PG_GETARG_BYTEA_PP(1);
+       char       *data = VARDATA_ANY(source2);
+       int32           len;
+       int32           result_len;
+       int32      *resultint = palloc0(sizeof(int32));
+       int32 conv_res = 0;
+       int32 result;
+       len = VARSIZE_ANY_EXHDR(source2);
+       result_len = len > sizeof(int32) ? sizeof(int32) : len;
+       reverse_memcpy((char *) resultint, data, result_len);
```

Comment:
is this expected?

Reply:
Yeah , I am converting varbinary to int4 in order to do division.